reinforcement_learning/common/sagemaker_rl/tf_serving_utils.py

```diff
@@ -1,9 +1,13 @@
 import ray
 import os
 import re
-from ray.rllib.utils.framework import try_import_tf
+import tensorflow
```

Comment:
This will cause a failure if user uses PyTorch container. Can you elaborate a bit on why would the original `try_import_tf` not work here?

Reply:
I think this module will not be run when PyTorch is the framework being used.
The `try_import_tf` import was from the `ray.rllib.utils.framework` module, which is not available in older versions of RLLib (<0.8.2). I found some failures due to that while testing, so I changed it here.

Reply:
Fair enough. Thanks Akash!